packages/now-build-utils/src/detect-builders.ts

```diff
@@ -106,5 +92,58 @@ export async function detectApiBuilders(
     });
 
   const finishedBuilds = builds.filter(Boolean);
-  return finishedBuilds.length > 0 ? (finishedBuilds as Builder[]) : null;
+  return finishedBuilds as Builder[];
+}
+
+// When zero config is used we can call this function
+// to determine what builders to use
+export async function detectBuilders(
+  files: string[],
+  pkg?: PackageJson | undefined
+): Promise<{
+  builders: Builder[] | null;
+  warnings: Warning[] | null;
+}> {
+  const builders: Builder[] = [];
+  const warnings: Warning[] = [];
+
+  // Detect all builders for the `api` directory before anything else
+  builders.push(...(await detectApiBuilders(files)));
```

Comment:
```suggestion
  const builders = await detectApiBuilders(files));
```

Why not assign here instead of destructuring?

Reply:
Done 👍